repos/fdbt-site/src/pages/products/productDetails.tsx

```diff
@@ -473,6 +478,35 @@ const createProductDetails = async (
         });
     }
 
+    if (ticket.type === 'flatFare' && 'pricingByDistance' in product) {
```

Comment:
do we not have a typeGuard for this? if not, we should create one. So we can do `if(isFlatFarePricedByDistanceTicket(ticket))`